workflow/kfdrc-somatic-variant-workflow.cwl

```diff
@@ -0,0 +1,800 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc-somatic-variant-workflow
+label: Kids First DRC Somatic Variant Workflow
+doc: |
+  # KFDRC Somatic Whole Genome Sequence Analysis Workflow
+
+  ![data service logo](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS9BnbvIsTkK3QlSGMDvlgu0tZQJ1q4crMvA-S3fcWfIq6y2d2Y)
+
+  This is the Kids First Data Resource Center (DRC) Whole Genome Sequencing (WGS) Somatic Workflow, which includes somatic variant calling, copy number variation (CNV), and structural variant (SV) calls.
```

Comment:
Same comment as before for this and readme, just slide it above the logo

Reply:
Say no more.